libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/PredictiveCacheControllerTest.kt

```diff
@@ -54,7 +54,10 @@ class PredictiveCacheControllerTest {
             PredictiveCache.createNavigationController(any())
         } just Runs
 
-        PredictiveCacheController(mockedLocationOptions, errorHandler)
+        PredictiveCacheController(
+            predictiveCacheGuidanceLocationOptions = mockedLocationOptions,
+            predictiveCacheControllerErrorHandler = errorHandler,
```

Comment:
Do we consider the need of using `Named Arguments` a breaking change? 🤔 

cc @mapbox/navigation-android

Reply:
I would say so, if we can avoid it, let's do that. This wouldn't be needed if we made `predictiveCacheGuidanceLocationOptions` the last constructor argument.

Reply:
Correct but we'd lose the SAM conversion ¯\\\_(ツ)\_/¯

Although probably better than "breaking" SemVer 👍

Reply:
> This wouldn't be needed if we made `predictiveCacheGuidanceLocationOptions` the last constructor argument.

This won't work either as would break clients already specifying `predictiveCacheControllerErrorHandler`. Opted to overload the constructor and pass `predictiveCacheLocationOptions` as `predictiveCacheGuidanceLocationOptions` when constructing `PredictiveCacheController` to retain backwards compatibility.